### PR TITLE
Fix imagePullSecrets generation in the service account template

### LINKED ISSUE #71: [k8s-service chart] Image pull secrets not mapped correctly for service account
`imagePullSecrets` is supposed to be a simple list of strings (the names of the secrets) but in the service account template it expects a map.

Once we add one item to the `imagePullSecrets` value we get:

```bash
Error: UPGRADE FAILED: error validating "": error validating data: ValidationError(ServiceAccount.imagePullSecrets[0]): invalid type for io.k8s.api.core.v1.LocalObjectReference: got "string", expected "map"
```

Service account:
https://github.com/gruntwork-io/helm-kubernetes-services/blob/master/charts/k8s-service/templates/serviceaccount.yaml#L16-L17

Deployment:
https://github.com/gruntwork-io/helm-kubernetes-services/blob/master/charts/k8s-service/templates/deployment.yaml#L259-L262

### PR DESCRIPTION
**Pull request by paul-pop**
Fixes #71

### CODE REVIEW

File: test/k8s_service_service_account_template_test.go [hunk 1]
```diff
@@ -22,30 +21,21 @@ func TestK8SServiceAccountCreateTrueCreatesServiceAccount(t *testing.T) {
 	t.Parallel()
 	randomSAName := strings.ToLower(random.UniqueId())
 
-	helmChartPath, err := filepath.Abs(filepath.Join("..", "charts", "k8s-service"))
-	require.NoError(t, err)
+	serviceaccount := renderK8SServiceAccountWithSetValues(
```
Comment (paul-pop) [inline test/k8s_service_service_account_template_test.go:24]: I've created the new helper function similar to the others and refactored the tests here.

**Comment by paul-pop** [comment]
@yorinasub17 is this something you can look at soon?

**Comment by yorinasub17** [comment]
Build passed, so will merge this in. Thanks again for your contribution!

**Comment by yorinasub17** [comment]
Released as `v0.1.2`. It should now be available at `helmcharts.gruntwork.io`.

**Pull request merged**